build/golang-build/Dockerfile

```diff
@@ -15,6 +15,9 @@ RUN apt-get update && \
     mkdir -p /go/src/github.com/google && \
     git clone --branch master https://github.com/google/protobuf /go/src/github.com/google/protobuf && \
     go get -u golang.org/x/lint/golint && \
+    go get github.com/fzipp/gocyclo && \
+    go get -u github.com/client9/misspell/cmd/misspell && \
+    go get -u github.com/gordonklaus/ineffassign && \
```

Comment:
The Travis CI build no longer uses the golang-build image. The build is done natively on the VM that Travis gives us. Please add these new dependencies to .travis.yml

Reply:
I'm seeing `  - env docker run -e GO111MODULE="on" -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" -e TRAVIS_BRANCH="$TRAVIS_BRANCH" -it -v `pwd`:/go/src/github.com/onosproject/simulators -w "/go/src/github.com/onosproject/simulators" onosproject/golang-build:stable build` in the .travis.yaml file meaning there is still some dependency on that image. Also there is no mention of lint in this .yaml file. I'm puzzled.

Reply:
done